weaviate/collection/collection.py

```diff
@@ -109,7 +110,26 @@ def delete(self, name: str) -> None:
         Parameters:
         - name: The name of the collection to delete.
         """
-        self._delete(_capitalize_first_letter(name))
+        ...
+
+    @overload
+    def delete(self, name: List[str]) -> None:
```

Comment:
Isn't this overdoing it a little bit? :)

I think having the union as input paramter is totally fine

Reply:
Heheh, yeh probably 😁 Its only benefit is showing different docstrings to the user depending which function they are calling since description of `.delete("class")` uses the singular number while `.delete(["one", "two"])` uses the plural